ludwig/explain/captum_ray.py

```diff
@@ -91,21 +93,57 @@ def explain(self) -> Tuple[List[Explanation], List[float]]:
         expected_values = []
         for attrs_ref in tqdm(attrs_refs, desc="Explain"):
             attrs = ray.get(attrs_ref)
-            for (total_attribution, feat_to_token_attribution) in attrs:
-                for i, (feature_attributions, explanation) in enumerate(zip(total_attribution, self.explanations)):
+            for total_attribution, feat_to_token_attributions, total_attribution_global in attrs:
```

Comment:
Not sure if there's any way getting around these nested for loops but I feel like this might blow up the explanation time by a bit right? If so, do we maybe want to compute the explanations on a sample of the dataset - or is that already happening - I can't remember